types/moo-indentation-lexer/index.d.ts

```diff
@@ -0,0 +1,42 @@
+import { Lexer, LexerState, Token } from "moo";
+
+export interface IndentLexer {
+    lexer: Lexer;
+    indentationType?: string;
+    newlineType?: string;
+    commentType?: string;
+    indentName?: string;
+    dedentName?: string;
+    enclosingPunctuations?: object;
+    separators?: string[];
+    state?: string;
+    enclosures?: Array<{ opening: string; indentationLevel: string }>;
+    indentations?: string[];
+    queuedTokens?: Token[];
+    queuedLines?: Token[][];
+    lastToken?: Token;
+}
+
+export class IndentationLexer implements Lexer {
```

Comment:
This isn't correct; the package does:

```js
module.exports = IndentationLexer
```

Which means that this declaration should have:

```
class IndentationLexer implements Lexer {}

export = IndentationLexer
```